### PR TITLE
Fix a Dart 2 type error in the Stocks example

### PR DESCRIPTION
**Pull request by jason-simmons**

### CODE REVIEW

**Comment by jason-simmons** [comment]
@chinmaygarde

**Review by chinmaygarde — APPROVED** [review]
lgtm. But I am not the best Dart reviewer.

**Comment by chinmaygarde** [comment]
Fixes https://github.com/flutter/flutter/issues/15632

File: examples/stocks/lib/stock_data.dart [hunk 1]
```diff
@@ -56,9 +56,9 @@ class StockData extends ChangeNotifier {
 
   bool get loading => _httpClient != null;
 
-  void add(List<List<String>> data) {
-    for (List<String> fields in data) {
-      final Stock stock = new Stock.fromFields(fields);
+  void add(List<dynamic> data) {
+    for (List<dynamic> fields in data) {
+      final Stock stock = new Stock.fromFields(fields.cast<String>());
```
Comment (cbracken) [inline examples/stocks/lib/stock_data.dart]: I'd be tempted to leave this code as is and do the coercion at the call site on line 89.

Comment (jason-simmons) [reply examples/stocks/lib/stock_data.dart]: afaik there isn't a way to do the cast there that passes the current static analysis config as well as the runtime checks

**Pull request merged**